README.md

```diff
@@ -61,27 +61,28 @@ jobs:
-| debug_print_variables            | false                      | Enables printing of input/environment variables within the docker container.                                                                                                                                                                  |
-| branch_uri                       | [none]                     | The URI to the branch from the SCM System                                                                                                                                                                                                     |
-| build_version                    | [none]                     | Version of application build artifacts                                                                                                                                                                                                        |
-| build_uri                        | [none]                     | URI to CI build info                                                                                                                                                                                                                          |
-| operating_environment            | [none]                     | System info regarding operating system, etc.                                                                                                                                                                                                  |
-| sarif                            | false                      | Enable Uploading the SARIF Report to GitHub.                                                                                                                                                                                                  |
-| gpat                             | [none]                     | GitHub Personal Access Token. Required to upload SARIF Report                                                                                                                                                                                 |
-| package_managers                             | [none]                     | List (comma separated) of Package Managers to filter manifest search. (Dart, Erlang, Homebrew, PHP, Java, Nuget, NPM, Python, Ruby, Rust.)                                                                                                                                                                               |
-| verbosity                             | INFO                     | Set logging verbosity level value (INFO/DEBUG)                                                                                                                                                                    |
-| verbose                             | false                     | Enable verbose logging |
+| Property | Description | Default |
```

Comment:
@SOOS-JAlvarez 
I'm not sure if this readme was generated automatically, let me know if it was & how to redo it

I wrote python to generate a new one, because the current readme was missing some inputs, like `mode`

```python
print('| Property | Description | Default |')
print('| --- | --- | --- ', end='')
for line in text.splitlines():
    leading_spaces = len(line) - len(line.lstrip(' '))
    line = line[leading_spaces:]
    if leading_spaces == 2:
        print('|\n|', line[:-1], end=' ')
    elif leading_spaces == 4:
        split_line = line.split(' ')
        if split_line[0] == 'description:' or split_line[0] == 'default:':
            print('|', ' '.join(split_line[1:]).strip('\''), end=' ')
```

Reply:
yeah I don't think I've generated them with python but that's ok, PR looks good, as the readme properties are the ones defined in the action.yml
https://github.com/soos-io/soos-sca-github-action/blob/main/action.yml

Reply:
👍🏻 thanks